hack/update-generated-clientset.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I am not a fan of these 50 liner bash scripts in every projects right now. But it's the best we have right now :-/